tests/wpt/web-platform-tests/html/webappapis/scripting/events/iframe-onload.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe id="blank_iframe"></iframe>
+
+<script>
+  var iframe_loads = 0;
+  function iframeOnLoad(t) {
+    iframe_loads++;
+
+    setTimeout(function() {
+      t.step(function() {
+        assert_true(iframe_loads === 1, "iframe.onload should be called exactly once.");
+      });
+      t.done();
+    }, 100);
+  }
+
+  async_test(function(t) {
+    t.step(function() {
+      document.getElementById("blank_iframe").onload = function() {
+        iframeOnLoad(t);
+      }
+    });
+  }, "IFrameSingleOnLoad");
+  document.getElementById('blank_iframe').contentWindow.location.reload();
```

Comment:
I had tested it on master and it did trigger two load events. However, I agree it's better to add a new iframe, since it's simpler and doesn't raise doubts. Will do!